### DISCUSSION TITLE
Group Assignment #2

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#8] [Assignments]
Use this thread to submit [Group Assignment #2](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_02/Materials/group_assignment.md) by having one person comment with the link to your memo.

**Comment by edmondywen** [comment]
Cynthia Orozco and Edmond Wen
https://docs.google.com/document/d/1GvA4k3uWjcz0QHtuldJoM6o4IRP7njExK3MbBG4-osU/edit

**Comment by abi-200** [comment]
Abita Venkatesh and Luigi Magdamit
https://docs.google.com/document/d/18AjnnQwUCjLvAJVxvDkmdPmsdU3LE8TydU8VBZvcWyg/edit?usp=sharing

**Comment by perezsara59** [comment]
Sara Perez and Harry Frahn
https://docs.google.com/document/d/1lpIwhBVANcFWM3tdo1rDt_4WZVeHrb9z_vwemLUaVmw/edit?usp=sharing

**Comment by ronnie-blanche** [comment]
Emily Suh and Ronnie Blanche Dinh
https://docs.google.com/document/d/1LNgnhffMUP-XEOb6GlL6KGhGzTfNeJfKmLY0abH1Nr0/edit?usp=sharing

**Comment by eandrusenki** [comment]
Liz Andrusenki and Irvin Ramirez
https://docs.google.com/document/d/1T3BBI2_Q_yezSUMO9ftBVT5UdLaeqBcvlR8cm_rNsb0/edit?usp=sharing
